doc/index.rst

```diff
@@ -35,19 +32,40 @@ interface toolkits.
    </div>
    <span class="clear_screenshots"></span>
 
-Matplotlib tries to make easy things easy and hard things possible.  You
```

Comment:
I personally really like this sentence (yes, this specific wording) and would be sad to see it go -- it really embodies the matplotlib philsophy to me ("making hard things possible" is what distinguishes matplotlib from many "simpler" wrappers, and "making easy things easy" means that you don't have to fiddle with the GUI toolkit and that you can output to a bunch of formats natively, as opposed to e.g. web-only (js) libs).

Reply:
I understand your point. I've thought for a while if I should keep this sentence.

**TL;DR I don't think the sentence is helpful or even suitable. But if people are attached to it, I would be willing to leave it in for now.**

I have three issues with it:

- "Matplotlib tries to ..." is too defensive. In a philosophy you don't try. If anything this would need to be

    Matplotlib makes easy things easy and hard things possible.

- Easy and hard things are highly vague and subjective.
  > "making easy things easy" means that you don't have to fiddle with the GUI toolkit and that you can output to a bunch of formats natively, as opposed to e.g. web-only (js) libs

  is your particular interpretation and focused on the output aspect.
  I've read it more broadly as "Things that I can easily describe should be easy to code."

  In it's generality, you could apply the statement to a lot of software (It appears to coined on the design of Perl). It's not a distinctive feature of matplotlib. I assume that most of the visualization libraries would claim that paradigm (with different interpretations of easy and hard).

- From a user perspective, IMHO we are not regarded as particularly simple. This is partly because we see core matplotlib as fundamental building blocks, and we have a limited number of high-level helpers to arrange them. This is a design choice, which is perfectly fine. I just don't think we should over-emphasize easy.

  It's not easy if I need several lines of code to make a simple plot with axes labels, title and legend. It's not easy if I have to calculate positions for [grouped bar charts](https://matplotlib.org/devdocs/gallery/lines_bars_and_markers/barchart.html). There are higher-level libraries that can claim "simple" much more.

Therefore, instead of "easy things easy and hard things possible", I hope to covey the two aspects more clearly and accurately with:
- Easy creation of numerous standard plots
- Customizable and extendable

Reply:
I'm not going to try to argue rationally here -- it is strictly a sentimental attachment to this sentence.  On the other hand, now that I stated my point, I'm not going to block its removal if there's agreement to do so.

(FWIW "Matplotlib makes easy things easy and hard things possible." is fine with me too, or perhaps "aims to ...".)

Reply:
I'm with @anntzer on this - "Matplotlib makes easy things easy and hard things possible" is the brand/mission/ideal. And yes many users may not agree that we make easy things easy but this is the sales pitch. I think your text with some copy editing underneath works as an explanatory thing.

Reply:
Ok, I would like to talk about the misson statement at some point. It is valuable to clarify that for ourselves and also to communicate it. - Will likely be a paragraph or two on a dedicated page.

For now, I will add the easy sentence back in.